From f837c7858b1d354094205b5f1b07e5b9c4a7df3d Thu, 12 Jul 2018 15:51:50 +0200
From: Pekka Vuorela <pekka.vuorela@jollamobile.com>
Date: Thu, 12 Jul 2018 15:51:33 +0200
Subject: [PATCH] use-mtab-instead-of-fstab

This is needed for Tracker to detect removable storage properly. Upstream 
by default already moved to libmount, but that seems to have problems with 
Tracker. Enabling commit (496cd8aac6402) does state that it returns slightly 
different results, namely with bind mounts.

diff --git a/gio/gunixmounts.c b/gio/gunixmounts.c
index 0035d57..6c26791 100644
--- a/gio/gunixmounts.c
+++ b/gio/gunixmounts.c
@@ -1049,7 +1049,7 @@
   GUnixMountPoint *mount_point;
   GList *return_list;
   
-  read_file = get_fstab_file ();
+  read_file = get_mtab_read_file ();
   
   file = setmntent (read_file, "r");
   if (file == NULL)
